### PR TITLE
chore: i18next-scanner

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
i18next-parser is limited in configuration and parsing.
@lmaillet tried i18next-scanner.

**What is the chosen solution to this problem?**
Use i18next-scanner.
* plural for free based on `count` 🎉 
* context : has to duplicate a translation to add a context (DeleteResource container)
* more complete configuration

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "commander": "^2.9.0",
     "cpx": "^1.5.0",
     "i18next-parser": "^0.13.0",
```
Comment (jmfrancois) [inline package.json]: could we remove i18next parser ?

Comment (jsomsanith) [reply package.json]: done thanks

File: packages/forms/i18next-scanner.config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	options: {
+		debug: true,
+		func: {
+			list: ['i18next.t', 'i18n.t', 't'],
```
[the remark this answers is not in the document]
Comment (lmaillet) [reply packages/forms/i18next-scanner.config.js]: Looks like having just `['t']` as the function list is enough. `props.t` and `i18n.t` are parsed using just `t`
```suggestion
			list: ['t'],
```

File: packages/containers/src/DeleteResource/DeleteResource.container.js [hunk 3]
```diff
@@ -110,6 +110,23 @@ export class DeleteResource extends React.Component {
 			className: 'btn-inverse',
 			onClickActionCreator: 'DeleteResource#cancel',
 		};
+
+		// Sorry for this duplication, but we need it because of the i18n scanner to create 2 keys
+		// DELETE_RESOURCE_MESSAGE and DELETE_RESOURCE_MESSAGE_female
+		let question;
+		if (this.props.female) {
```
[the remark this answers is not in the document]
Comment (jsomsanith) [reply packages/containers/src/DeleteResource/DeleteResource.container.js]: Works like a charm, thank you

**Comment by lmaillet** [comment]
FYI, I've found a small bug about whitespaces when parsing `<Trans>` component.
I've opened an issue: https://github.com/i18next/i18next-scanner/issues/126

File: packages/forms/i18next-scanner.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsomsanith) [reply packages/forms/i18next-scanner.config.js]: You're right, I copied the example and removed the things we don't need, seems like I forgot this one.
Thanks

File: packages/containers/src/DeleteResource/DeleteResource.container.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (jsomsanith) [reply packages/containers/src/DeleteResource/DeleteResource.container.js]: Hmm no in fact it doesn't work :D
The extraction works but we can't set the context after t() call. You have a string instead

[the remark this answers is not in the document]
Comment (annilsson) [reply packages/containers/src/DeleteResource/DeleteResource.container.js]: You're right, I forgot about actually using the code too 😄

**Pull request merged**